deploy/scheduler/config_map.yaml

```diff
@@ -38,6 +38,6 @@ data:
         "timeoutSeconds": 5
       },
       "migrationDeletionRetrySeconds": 5,
-      "doMigration": false,
+      "doMigration": true,
```

Comment:
> Now that it's opt-in, we can *also* set the global config setting to
default to true. Still worth keeping for now, at the very least as a
"break glass in case of emergency" button.


Does the commit message relates to this change?

Reply:
Yep, that's what that refers to

Reply:
I'd rewrite can -> do and mention a name of a field to be explicit. Like this:

> Also set the global doMigration to default to true. 
> Still worth keeping for now, at the very least as a
> "break glass in case of emergency" button.

Reply:
done